h/security/identity.py

```diff
@@ -1,6 +1,60 @@
 from dataclasses import dataclass
+from typing import List, Optional
 
-from h.models import AuthClient, User
+from h.models import AuthClient, Group, User
+
+
+@dataclass
+class LongLivedGroup:
```

Comment:
This mirrors the `Token`, `LongLivedToken` in `h.services.auth_token` which is a similar idea.

Previously our `LongLivedToken` was called `Token` which is horribly confusing and causes all sorts of namespace clashes in the tests and elsewhere.

I'd like it to be very obvious in stack-traces and the like that this isn't a normal `Group` object.

Reply:
Just `Token` etc will be confusing yeah. Without reading the docstring I'd assume `LongLivedToken` is about expiration dates or similar but I don't have a better name `ServiceToken` nah `IdentityToken`? meh.